python/spear/config/default_config.vehicle.yaml

```diff
@@ -0,0 +1,46 @@
+#
+# Copyright(c) 2022 Intel. Licensed under the MIT License <http://opensource.org/licenses/MIT>.
+#
+
+VEHICLE:
+  VEHICLE_PAWN:
+    SKELETAL_MESH: "/Vehicle/OpenBot/Meshes/OpenBot_Mesh.OpenBot_Mesh"
+    ANIM_INSTANCE: "/Vehicle/OpenBot/Meshes/OpenBot_Animation.OpenBot_Animation_C"
+
+    CAMERA_COMPONENT:
+      POSITION_X: -60.0            # Camera relative position in [cm]
+      POSITION_Y: 0.0              # Camera relative position in [cm]
+      POSITION_Z: 30.0             # Camera relative position in [cm]
+      PITCH: -10.0                 # Camera relative orientation in [°]
+      YAW: 0.0                     # Camera relative orientation in [°]
+      ROLL: 0.0                    # Camera relative orientation in [°]
+      FOV: 70.0                    # Camera field of view in [°]
+    
+    IMU_COMPONENT:
+      POSITION_X: 8.0              # IMU relative position in [cm]
+      POSITION_Y: 0.0              # IMU relative position in [cm]
+      POSITION_Z: 10.0             # IMU relative position in [cm]
+      PITCH: 0.0                   # IMU relative orientation in [°]
+      YAW: 0.0                     # IMU relative orientation in [°]
+      ROLL: 0.0                    # IMU relative orientation in [°]
+
+  VEHICLE_COMPONENT:
+    MASS: 11.2                     # Mass of the vehicle chassis in [kg]
+    DRAG_COEFFICIENT: 1.0          # DragCoefficient of the vehicle chassis
+    CHASSIS_WIDTH: 15.0            # Chassis width used for drag force computation in [cm]
+    CHASSIS_HEIGHT: 15.0           # Chassis height used for drag force computation in [cm]
+    ENABLE_SUSPENSION: True
+    ENABLE_WHEEL_FRICTION: True
+    ENABLE_LEGACY_WHEEL_FRICTION: False
+
+  VEHICLE_WHEEL:
+    WHEEL_MASS: 0.033                 # Mass of this wheel in [kg]
```

Comment:
I think all of the comments in this file can be aligned with each other for maximum readability

Reply:
Updated